oop-1-class-constructor/index.md

```diff
@@ -19,7 +19,7 @@ nav_exclude: true
 
 ## Recap & Assignment check
 
-Let's look into the assignment from lesson 6
+Let's look into the assignment from lesson 3
 
 [Google Slides](https://docs.google.com/presentation/d/1M_Eeyv9fA4oJda54TCXdyhBqFoclJgqYenxT7GqkXwo/embed)
 
```

Comment:
are we intending to add slides or it's by mistake? we didn't use slides in previous sessions

Reply:
The slides are publicly available to be used. Feel free to use them if you wish. I recommend that you do for this particular session.